### DISCUSSION TITLE
[fixed] Umlauts are not properly displayed in sidebar

### DISCUSSION THREAD

**Discussion by kakra** [KDE/ghostwriter#664] [General]
*The issue tracker is closed, so I'll leave a message here. Feel free to just transfer to your internal list and close, thanks.*

If using umlauts (or other special utf-8 chars), these are not properly displayed in the sidebar:

![Screenshot_20210915_114519](https://user-images.githubusercontent.com/17684/133412579-026b530f-b3ac-43d4-9040-30619f83e5f2.png)

Thanks for a great editor, btw.

**Comment by wereturtle** [comment]
They do with the latest version.  What version are you on?

**Reply by kakra** [reply]
Okay, actually that's interesting: On my other PC they DO work, but not on my office PC. I'll check the version later when I am there. This PC has 2.0.2 that I checked just now.

**Reply by kakra** [reply]
Actually, you're right. For some reason, the ghostwriter version of my office PC didn't update. It works now after updating it. Sorry for the noise.

*PS: It would actually be useful if the issue tracker could exist in some archived read-only mode, so I could have checked if this was already supported. But still, thanks for the prompt help.*
